int/tests/test_credentialdefinitions.py

```diff
@@ -0,0 +1,101 @@
+"""Credential Definition Tests"""
+import asyncio
+import pytest
+
+
+@pytest.fixture(scope="module")
+async def create_schema(connection, endorser_did):
+    """Schema factory fixture"""
+
+    async def _create_schema(version):
+        return await connection.send_and_await_reply_async(
+            {
+                "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/admin-schemas/0.1/send-schema",
+                "schema_name": "Test Schema",
+                "schema_version": version,
+                "attributes": ["attr_1_0", "attr_1_1", "attr_1_2"],
+                "return_route": "all",
+            }
+        )
```

Comment:
We should prefer using the backchannel to perform this operation instead of the admin-schema protocol. That way, if an admin-schema issue arises in the future, it won't interfere with the admin-cred-def protocol as well.

Reply:
The backchannel method to use is `acapy_backchannel.api.publish_schema` where the `json_body` is a `SendSchemaRequest` from `acapy_backchannel.models`.